logic/version.go

```diff
@@ -9,7 +9,7 @@ import (
 
 const MinVersion = "v0.17.0"
 
-// IsVersionCompatible checks that the version passed is compabtible (>=) with MinVersion
+// IsVersionComptatible checks that the version passed is compabtible (>=) with MinVersion
```

Comment:
there is a typo in the func name, rather than changing the comment func name should be changed